source/issues-and-help.txt

```diff
@@ -0,0 +1,50 @@
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+We are lucky to have a vibrant MongoDB C#/.NET community
+that includes users with varying levels of experience using
+the driver. We find the quickest way to get support for general
+questions is through the `MongoDB Community Forums
+<https://www.mongodb.com/community/forums>`__.
+
+To learn more, refer to our :manual:`support channels </support>`.
+
+Bugs / Feature Requests
+-----------------------
+
+To report a bug or to request a new feature in the C#/.NET driver,
```

Comment:
Suggestion:

Since this repo uses [this source constant](https://github.com/mongodb/docs-csharp/blob/master/snooty.toml#L18) for the name of the driver, it is helpful to use in every instance. Otherwise, if the name ever changes, someone will need to search through the entire set of docs to find and replace rather than changing the source constant only.

Reply:
I understood this comment to mean that this page should follow the order of .NET/C# where mentioned.